### PR TITLE
feat(session-table): Change lazy session fields to match property names

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem
I'm pulling this out of a much larger PR as it's a pretty standalone change and I wanted to make it easier to review. It doesn't really solve a problem on its own, but it does allowing introspection of the lazy sessions table to figure out what session properties are available.

## Changes

Rename the columns in `SessionsTable` to match the naming of properties.

It's worth calling out here that `session.duration` in hogql will no longer work. I don't think this is a problem given
* hogql is in beta
* the fix is simple, rename to session.$session_duration
* hogql session.duration already changed recently, to using the session table. It's not stable

However it'd be pretty easy to add an alias if we felt strongly the other way.

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Updated existing tests

### CODE REVIEW

File: posthog/hogql/property.py [hunk 1]
```diff
@@ -163,10 +163,6 @@ def property_to_expr(
         properties_field = ast.Field(chain=chain)
         field = ast.Field(chain=chain + [property.key])
 
-        if property.type == "session" and property.key == "$session_duration":
```
Comment (robbie-c) [inline posthog/hogql/property.py:166]: No longer needed!

**Review by mariusandra — COMMENTED** [review]
In general LGTM, though why not just `duration` or `$duration`? 

I even see code like 

```sql
        min(sessions.$start_timestamp) as $start_timestamp,
        any(sessions.$session_duration) as session_duration,
        any(sessions.$pageview_count) as session_pageview_count,
        any(sessions.$autocapture_count) as session_autocapture_count,
```

... so removing `session_` would add consistency 🤔 

As for backwards compatibility, we can check if we need it. If someone has a HogQL insight saved that uses `session.duration` anywhere, it's not too much trouble for us to search and replace them to the new key. There might be dozens of users we'll otherwise needlessly frustrate.

I have nothing against a `duration` alias, or just keeping it without the `$` as well 🤷

**Comment by robbie-c** [comment]
It's because I want the table names to match up with the property names, and the property with $session_duration will be much more common, I had assumed it would be a pain to track down and find all of the usages.

It hadn't occurred to me that it's easy to find/replace people's insights! That's good to know.

IMO the best option is to keep the rename, but add the alias to duration so that we don't need to update people's insights right now.

**Comment by mariusandra** [comment]
> IMO the best option is to keep the rename, but add the alias to duration so that we don't need to update people's insights right now.

That works for me. Best not to break queries if we can avoid it.

**Pull request merged**